content/automate/ManageIQ/PhysicalInfrastructure/CiscoIntersight/GetMethods/GetServerProfileTemplates.class/__methods__/get_server_profile_templates.rb

```diff
@@ -0,0 +1,14 @@
+dialog_field = $evm.object
+dialog_field["sort_by"] = "value"
+dialog_field["sort_order"] = "ascending"
+dialog_field["data_type"] = "string"
+dialog_field["required"] = true
+
+selected_ems_id = $evm.root.attributes['dialog_ems']
+selected_ems = $evm.vmdb(:ext_management_system).where(:id=>selected_ems_id).first
```

Comment:
If you have a single selected ems ID then you don't need to use `.where` you should use `.find()` or `.find_by(:id =>)` depending on if you want a `RecordNotFound` exception raised vs handle `nil` if not found.

Reply:
fixed